views/layout.hbs

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/stylesheets/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>IronBeers</title>
+</head>
+<body>
+    <nav>
+        <div>
+            <a href="/">Home</a>
+            <a href="/beers">Beers</a>
+            <a href="/random-beer">Random Beer</a>
+        </div>
+    </nav>
+    {{{ body }}}
+    <footer>
+        <div>
+            <p>Developed by Gergö :)</p>
+        </div>
+    </footer>
```

Comment:
Your footer is covering some content - the brewer's tips. Maybe you can try to fix that.